tools/nlp-toolkit/src/NlpToolkit.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+	<div class="NlpToolkit">
+
+		<tool-markdown :markdown="`
+This is a tool to to perform NLP operations using the library [compromise](https://www.npmjs.com/package/compromise).
+Enter text below:
+		`"/>
+		<section class="NlpToolkit__content">
+			<tool-code
+				ref="NlpToolkit__input"
+				class="NlpToolkit__content__half"
+				v-model="inputText"
+				:autoSize="true"
+				:options="codeOptions"/>
+
+			<p v-text="resultText"/>
+		</section>
+
+		<tool-taskbar v-if="!!inputText">
+
+			<tool-button
+				class="NlpToolkit__button"
+				size="lg"
+				v-text="'Compute'"
+				@click.native="onComputeClick"/>
+
+			<b-form-group
+				class="NlpToolkit__button"
+				label="NLP Operation"
+				label-for="NlpTookit__operation"
+				label-size="sm">
+				<b-form-select
+					id="NlpTookit__operation"
+					v-model="selectedOperation"
+					:options="nlpOptions"/>
+			</b-form-group>
+
+			<tool-button
+				size="sm"
+				class="NlpToolkit__button"
+				variant="secondary"
+				v-text="'Reset'"
+				@click.native="reset"/>
+
+		</tool-taskbar>
+
+	</div>
+</template>
+<script>
+import nlp from 'compromise';
+
+const nlpMethods = {
+
+	toPastTense: text => {
+		const doc = nlp(text);
+		doc.verbs().toPastTense();
+		return doc.text();
+	},
+
+	toFutureTense: text => {
+		const doc = nlp(text);
+		doc.verbs().toFutureTense();
+		return doc.text();
+	},
+
+	negate: text => {
+		const doc = nlp(text);
+		doc.verbs().toNegative();
+		return doc.text();
+	}
+};
+
```

Comment:
Is there a reason why these functions are written up here and not in the methods section below?